### PR TITLE
Fix Python 3 compatibility

### PR DESCRIPTION
**Pull request by vmarkovtsev**

### CODE REVIEW

**Comment by vmarkovtsev** [comment]
I signed it!

File: gfm/__init__.py [hunk 1]
```diff
@@ -2,6 +2,10 @@
 # for details. All rights reserved. Use of this source code is governed by a
 # BSD-style license that can be found in the LICENSE file.
 
+
```
Comment (nex3) [inline gfm/__init__.py]: Remove the additional whitespace.

File: gfm/__init__.py [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+import os
+import sys
+sys.path.insert(0, os.path.dirname(__file__))
```
Comment (nex3) [inline gfm/__init__.py]: This seems like a problematic global modification. Is there no way to do relative imports that works across Python 2 and 3?

File: gfm/autolink.py [hunk 3]
```diff
@@ -39,6 +39,6 @@ class AutolinkExtension(markdown.Extension):
     def extendMarkdown(self, md, md_globals):
         url_re = r'(?i)\b((?:(?:ftp|https?)://|www\d{0,3}[.])(?:[^\s()<>]+|' + \
             r'\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()' + \
-            r'<>]+\)))*\)|[^\s`!()\[\]{};:' + r"'" + ur'".,<>?«»“”‘’]))'
+            r'<>]+\)))*\)|[^\s`!()\[\]{};:' + r"'" + r'".,<>?«»“”‘’]))'
```
Comment (nex3) [inline gfm/autolink.py]: Won't removing `u` here break Python 2?

**Pull request closed without merging**